### ISSUE TITLE
Unable to Open Conversations in Messaging App with TalkBack Screen Reader

### ISSUE THREAD

**Issue by CarlosM10011** [GrapheneOS/os-issue-tracker#1488]
I'm running the latest stable version of Graphene OS on a Pixel 5a and am using the stock version of TalkBack. Double-tapping on a conversation in the Messaging App does not do anything; the conversation will not open. Turning off TalkBack and tapping on an existing conversation does work, however.

# To Reproduce:

- Turn on TalkBack in settings (install a tts engine if you don't have one installed first)
- Open the stock sms messaging app
- Try and double tap on a conversation.

Let me know if I should provide any additional debugging info.

**Comment by thestinger** [comment]
I recommend using a different messaging app. We aren't able to actively develop one, and this AOSP sample app isn't actively maintained. We can't track all the issues with it here.

**Comment by thestinger** [comment]
AOSP Messaging issue.
